src/App.js

```diff
@@ -12,10 +14,17 @@ function App() {
     <>
       <header className="App-header">
         <div className="logo-title">
+          <RiFlickrFill className="icon1" />
+          <RiFlickrLine className="icon2" />
```

Comment:
##### Dear @JuanDBta, for the layout of the project we are required to follow the design guideline closely and your design is missing some points despite it looking great we still need to make some changes. 
- [ ] For the navbar, some of the design belongs to the end device (mobile) we are going to use to open the page and should not be part of the webpage. this includes `the wifi indicator`, `the battery indicator `, `The network indicator`, and more. Please ignore this part of the design. 
- [ ] Notice the image on the original design are background images, the text goes over the image, and in your design, this is an image of their own and the text is on the other side. This is against the design guidelines. 
- [ ] On the original design, the items are aligned in `two columns` in a `single row` each containing their own item. In your carse every row contains a single column, Please follow the design guideline and arrange them in two columns. 


### Your design 
![Screenshot from 2023-07-22 12-41-28](https://github.com/JuanDBta/met-art-app/assets/69077061/db7ba668-051d-46f3-85b2-7a46d7fe23d0)

### The original design 
![Screenshot from 2023-07-22 12-43-56](https://github.com/JuanDBta/met-art-app/assets/69077061/de7b4f85-5c2b-4685-8027-552ea04de429)

Reply:
- Well resolved 👍🏽